### DISCUSSION TITLE
Can I change the Idle Notifier notification sound to a custom .wav file of my choosing?

### DISCUSSION THREAD

**Discussion by Schaefer6** [runelite/runelite#17074] [Q&A] [answered]
I've tried different 'solutions' online for changing this notification sound to no avail—nothing is working. just wondering if it's possible.

**Comment by geheur** [comment]
what exactly did you do?
you have to put the notification.wav file in `.runelite` and set the sound to "custom" in "runelite" plugin.

**Reply by Schaefer6** [reply]
AppData > Local > RuneLite? (I placed the .wav file in here and it did not change the sound) and I did switch it to 'custom' in the settings dropdown

**Reply by geheur** [reply, marked as answer]
no, %userprofile%\\.runelite i.e. c:/users/username/.runelite

**Reply by Schaefer6** [reply]
thank you! it's working now.
